saml-module/latest/install/index.md

```diff
@@ -77,22 +103,37 @@ If you have not enforced SAML for a specific service provider, you can use the o
 
         Set the file permissions accordingly to limit who can read it.
 
-    3. Generate a SAML keystore metadata file in the same location as the keystore and add the following content:
+    3. (Recommended) To use the latest keystore configuration method, set the following as JVM properties:
 
        ```bash
-       aliases=my-saml-key
-       keystore.password=change-me
-       my-saml-key.password=change-me
+       ...
+       saml-keystore.aliases=my-saml-key
+       saml-keystore.password=change-me
+       saml-keystore.my-saml-key.password=change-me
+       saml-keystore.my-saml-key.algorithm=AES
        ```
 
-       Set the file permissions accordingly to limit who can read it.
+    4. (Deprecated) To continue using the old keystore configuration method:
 
-    4. Set the following values in the `alfresco-global.properties` file:
+        1. Generate a SAML keystore metadata file in the same location as the keystore and add the following content:
```

Comment:
this part should remain like it was before..I believe it was step 6.3:
aliases=my-saml-key
keystore.password=change-me
my-saml-key.password=change-me
These ones should be in the metadata file

Reply:
Reverted update in the next commit